assets/scss/components/_reminders.scss

```diff
@@ -0,0 +1,131 @@
+/*
+Reminder List
+
+Display an Unordered List of reminder items with time formatted in a new line.
+
+Markup:
+<ul class="reminder-list">
+  <li>Buy Envelope</li>
+  <li>Buy Stamp</li>
+  <li>Post form by <time>5:00PM</time></li>
+</ul>
+
+
+Styleguide Reminders.reminder-list
```

Comment:
Please capitalise and you don't need the `reminder` as you are already in Reminders
`Reminders.reminder-list`
becomes
`Reminders.List`